src/hermes_claude_agent_sdk/parity/v4_cli.py

```
@@ -0,0 +1,324 @@
+    if _paths_overlap(output_path, (packets_source, receipts_source)):
+        raise V4CLIError("output must be separate from input artifacts")
+    try:
+        contract = load_v4_contract(contract_source)
+    except V4ContractViolation:
+        raise
+    packet_paths, trials = _load_packets(packets_source)
+    receipts = _load_receipts(receipts_source, packet_paths)
+    if set(receipts) != {item.name for item in packet_paths}:
+        raise V4CLIError("ownership receipts do not match the packet set exactly")
+    bound = []
+    for packet_path, trial in zip(packet_paths, trials):
```

Ruff (0.16.3)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)